ui/src/components/Views/Update/UpdateDialog.tsx

```diff
@@ -51,16 +56,29 @@ export const UpdateDialog = ({ open, onClose }: Props): ReactElement => {
     <Dialog open={open} onClose={onClose}>
       <DialogContent>
         <Box m={2} width={500} height={400}>
-          {logs.map(log => (
+          {
+            logs.map(log => (
+              <>
+                <Typography>
+                  {log}
+                </Typography>
+                <br />
+              </>
+            ))
+          }
+          {
+            logs.length === 0 &&
             <>
               <Typography>
-                {`${log}`}
+                Updating docker images
```

Comment:
```suggestion
                Updating Docker images
```

Reply:
@whummer you should change it also here https://github.com/localstack/localstack/blob/master/localstack/cli/localstack.py#L373